### PR TITLE
Added touch API

### PR DESCRIPTION
**Pull request by zsbzsb**

### CODE REVIEW

File: src/SFML/Window/Touch.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+sfVector2i sfTouch_getPosition(unsigned int finger, const sfWindow* relativeTo)
+{
+    sf::Vector2i position;
+
+    if (relativeTo)
+        position = sf::Touch::getPosition(finger, relativeTo->This);
+    else
+        position = sf::Touch::getPosition(finger);
+
+    return {position.x, position.y};
```
Comment (LaurentGomila) [inline src/SFML/Window/Touch.cpp]: This is C++11, see the other PR for the correct code.

**Comment by LaurentGomila** [comment]
You forgot the touch events.

**Comment by zsbzsb** [comment]
Done, touch events will come in another PR.

**Comment by zsbzsb** [comment]
Fixing the conflicts.... :P

**Comment by zsbzsb** [comment]
And ready to merge.

**Pull request merged**